From 96878b52ea9c27220698c13ff58739bbcf90330e Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Tue, 30 Oct 2018 10:31:35 -0500
Subject: [PATCH 3/8] windows fixes

---
 plugins/NTMakefile | 2 +-
 win32/common.mak   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/NTMakefile b/plugins/NTMakefile
index 3bf715a..bab0352 100755
--- a/plugins/NTMakefile
+++ b/plugins/NTMakefile
@@ -39,7 +39,7 @@ SQLITE_LIBS = /libpath:$(SQLITE_LIBPATH) libsqlite.lib
 !IF "$(SQL)" == "SQLITE3"
 PLUGINS_EXT=$(PLUGINS_EXT) saslSQLITE.dll
 SQL_FLAGS= $(SQLITE_INCLUDES3) /DHAVE_SQLITE3=1
-SQLITE_LIBS = /libpath:$(SQLITE_LIBPATH3) libsqlite3.lib
+SQLITE_LIBS = /libpath:$(SQLITE_LIBPATH3) sqlite3.lib
 !ENDIF
 
 PLUGINS=saslANONYMOUS.dll \
diff --git a/win32/common.mak b/win32/common.mak
index 194dfe9..3b4dc6d 100644
--- a/win32/common.mak
+++ b/win32/common.mak
@@ -45,7 +45,7 @@ TARGET_WIN_SYSTEM=0
 
 # prefix variable is currently only being used by install target
 !IF "$(prefix)" == ""
-prefix=C:\CMU
+prefix=$(LIBRARY_PREFIX)
 !IF "$(VERBOSE)" != "0"
 !MESSAGE Default installation directory is $(prefix).
 !ENDIF 
-- 
2.23.0

