From 94ff238911ac1b6fc07000b357cc3e9c0f62e64a Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Thu, 24 Sep 2020 19:28:23 +0200
Subject: [PATCH 7/8] win32: Skip building documentation (doc folder has no
 NTMakefile)

---
 NTMakefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/NTMakefile b/NTMakefile
index d3aac8a..7d21634 100644
--- a/NTMakefile
+++ b/NTMakefile
@@ -46,7 +46,9 @@
 
 SUBDIRS=lib plugins utils sample
 INCSUBDIRS=include sasldb win32\include
-DOCSUBDIRS=doc
+# No NTMakefile in this directory
+# DOCSUBDIRS=doc
+DOCSUBDIRS=
 
 all-first: all
 
-- 
2.23.0

